lib/cucumber_factory/factory.rb

```diff
@@ -200,6 +201,9 @@ def resolve_scalar_value(world, model_class, attribute, value)
           value = value.to_i
         elsif matches_fully?(value, VALUE_DECIMAL)
           value = BigDecimal(value)
+        elsif matches_fully?(value, VALUE_FILE)
+          path = File.realpath("./#{file_value_to_path(value)}")
```

Comment:
@niklas-hasselmeyer could you think of an alternative to `realpath` here?
I figured that this will prevent the use of symlinks, which might be a drawback for some uses.